<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryPageClass</class>
 <widget class="QWidget" name="QueryPageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QueryPage</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="m_tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>按领取人</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>54</width>
               <height>12</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>54</width>
               <height>12</height>
              </size>
             </property>
             <property name="text">
              <string>领取人:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="m_getPeople_detail">
             <property name="minimumSize">
              <size>
               <width>121</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>121</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="m_exportPeople">
             <property name="text">
              <string>导出数据</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="m_getPeople_tableView"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>按物品名称</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>54</width>
               <height>12</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>54</width>
               <height>12</height>
              </size>
             </property>
             <property name="text">
              <string>物品名称:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="m_getProName_detail">
             <property name="minimumSize">
              <size>
               <width>151</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>151</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="m_exportProName">
             <property name="text">
              <string>导出数据</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="m_getProName_tableView"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>按购买时间</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>54</width>
               <height>12</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>54</width>
               <height>12</height>
              </size>
             </property>
             <property name="text">
              <string>起始日期:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="m_buytimeStart">
             <property name="minimumSize">
              <size>
               <width>141</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>141</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>54</width>
               <height>12</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>54</width>
               <height>12</height>
              </size>
             </property>
             <property name="text">
              <string>终止日期:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="m_buytimeEnd">
             <property name="minimumSize">
              <size>
               <width>141</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>141</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="m_exportBuyTime">
             <property name="text">
              <string>导出数据</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="m_buytime_tableView"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>按领取时间</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="minimumSize">
              <size>
               <width>54</width>
               <height>12</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>54</width>
               <height>12</height>
              </size>
             </property>
             <property name="text">
              <string>起始日期:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="m_gettimeStart">
             <property name="minimumSize">
              <size>
               <width>141</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>141</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="minimumSize">
              <size>
               <width>54</width>
               <height>12</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>54</width>
               <height>12</height>
              </size>
             </property>
             <property name="text">
              <string>终止日期:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="m_gettimeEnd">
             <property name="minimumSize">
              <size>
               <width>141</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>141</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="m_exportGetTime">
             <property name="text">
              <string>导出数据</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="m_gettime_tableView"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
